app/src/main/java/ingsw/group1/findmyphone/MessageBuilder.java

```diff
@@ -0,0 +1,101 @@
+package ingsw.group1.findmyphone;
+
+import android.location.Location;
+
+import androidx.annotation.NonNull;
+
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSPeer;
+
+import ingsw.group1.findmyphone.alarm.AlarmManager;
+import ingsw.group1.findmyphone.location.LocationManager;
+
+/**
+ * Builder class useful for creating Request/Response messages
+ *
+ * @author Turcato
+ */
+public class MessageBuilder {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MessageBuilder' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5068369)